src/libraries/Common/src/Interop/Windows/Netapi32/Interop.DsGetDcNext.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Netapi32
+    {
+        [LibraryImport(Libraries.Netapi32, EntryPoint = "DsGetDcNextW", StringMarshalling = StringMarshalling.Utf16)]
```

Comment:
Do we always specify `StringMarshalling` when string is not used?

Reply:
From glancing at a few existing files, it would appear not - I'm happy to go through and remove the redundant ones touched in this PR? Likewise with any remaining redundant `EntryPoint` properties?

Reply:
Keeping consistency with existing code under Interop folder should be preferred.

Reply:
Roger that, done in f4dfe7f7271cea758098f065284cc55a8517f149 and f78219a77aaf13f1ebfae79af7d43d806b697daf